docs/guides/3-javascript.md

```diff
@@ -1,28 +1,53 @@
 # Spectral in JavaScript
 
-The Spectral CLI is a thin wrapper around a JavaScript (TypeScript) API, which can be used independently to do all the same things outside of the CLI.
+The Spectral CLI is a thin wrapper around a JavaScript (TypeScript) API, which can be used independently to do all the same things outside of the CLI, such as linting YAML and JSON documents from a string or an object.
 
-Assuming it has been installed as a Node module via NPM/Yarn, it can be used to lint YAML and JSON documents from a string, or from an object.
+## Prerequisites
 
-First of all, in order to consume the JS API you need to install the appropriate package(s).
+To use the Spectral JS API, you need to install the appropriate package.
+
+For npm users:
 
 ```bash
 npm install -g @stoplight/spectral-core
 ```
 
-If you are a Yarn user:
+For Yarn users:
 
 ```bash
 yarn global add @stoplight/spectral-core
 ```
 
-## Linting a YAML String
+### CommonJS and ESM
+
+The examples on this page are written in ESM. If you're using CommonJS, you have to import an additional module:
```

Comment:
This is only for custom functions. I don't think we cover custom functions on this page, thus I think we can remove it